src/main/java/com/google/api/codegen/transformer/php/PhpGapicSurfaceTransformer.java

```diff
@@ -292,4 +434,27 @@ private void addApiImports(GapicInterfaceContext context) {
 
     return apiMethods;
   }
+
+  private String generateConfigOutputPath(GapicInterfaceContext context, String name) {
+    GapicInterfaceConfig interfaceConfig = context.getInterfaceConfig();
+    String outputPath =
+        pathMapper.getOutputPath(
+            context.getInterfaceModel().getFullName(), context.getProductConfig());
+    return outputPath.substring(0, outputPath.length() - 5)
+        + "/resources/"
+        + Name.upperCamel(interfaceConfig.getInterfaceModel().getSimpleName())
```

Comment:
We should probably push naming logic like this into the SurfaceNamer class, even if we need to create a new unimplemented method in the base class and implement it in the PhpSurfaceNamer.